windows/runner/win32_window.cpp

```diff
@@ -117,7 +117,7 @@ bool Win32Window::CreateAndShow(const std::wstring& title,
   double scale_factor = dpi / 96.0;
 
   HWND window = CreateWindow(
-      window_class, title.c_str(), WS_OVERLAPPEDWINDOW | WS_VISIBLE,
+      window_class, title.c_str(), WS_OVERLAPPEDWINDOW,
```

Comment:
This changed works great on Linux, but of course has issues on Windows:

- Even though the window is hidden at first, it still steals focus from the active application.
- Starting with `WS_EX_NOACTIVATE` does not steal focus, but the window then can't be interacted with and there seems no way to undo this if it was created with this flag.
- Starting the window minimized works to not steal focus, but then programmatically restoring the window (for example if Nyrna was not run with the toggle flag) causes the entire app to enter "Not Responding".

All research seems to say that launching the app without stealing focus is only possible with `WS_EX_NOACTIVATE`, which definitely doesn't work here.

Possible solution (requiring far too much work): Nyrna launches on system start-up and remains hidden. The app can be called from the system tray, and the hotkey can ask the running instance of Nyrna to suspend the active window via inter-process communication -- presumably there is something like D-Bus to facilitate this. Alternatively even more work is to use the `win32` package to implement the hotkey and systray natively in Nyrna for Windows 😞